token-lending/program/src/processor.rs

```diff
@@ -1897,7 +1905,7 @@ fn _liquidate_obligation<'a>(
         token_program: token_program_id.clone(),
     })?;
 
-    Ok(withdraw_amount)
+    Ok((withdraw_amount, bonus_rate))
```

Comment:
bonus rate needs to be returned because it can't be recomputed, and it's needed to figure out protocol cut of liquidation bonus later